boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/liberty/LibertyPackageMojo.java

```diff
@@ -68,6 +70,8 @@ public void execute() throws MojoExecutionException {
         springBootVersion = MavenProjectUtil.findSpringBootVersion(project);
 
         boosterParent = new BoosterPacksParent();
+        
+        libertyServerPath = projectBuildDir + "/liberty/wlp/usr/servers/" + libertyServerName;
```

Comment:
I think Charles was suggesting that the quoted string here should be a constant.

Reply:
Not the whole quoted string - just the install directory (wlp folder) - which is `         libertyServerPath = projectBuildDir + "/liberty/wlp"`. Setting that as a constant and referencing file paths relative to the install directory variable would make the code more readable and flexible (in case the install directory changes or is configurable in the future).

Reply:
hmm.  we only ever use that quoted string in reference to the full path to the server (i.e. projectBuildDir + string + libertyServerName).  I think it might be cleaner to have the full libertyServerPath variable defined to avoid that same string concatenation in multiple places.

Reply:
Ok, that's fine. If we need to change it later we can.